xml/common_intro_feedback_i.xml

```diff
@@ -16,66 +16,69 @@
-    </para>
-    <para os="osuse">
-     To report bugs &productname;, go to <link
-     xlink:href="https://bugzilla.opensuse.org/"/>, log in, and click
-     <guimenu>New</guimenu>.
+     To open a service request, you need a subscription at &scc;. Go to
+     <link xlink:href="https://scc.suse.com/support/requests"/>, log in, and click
+     <guimenu>Create New</guimenu>.
     </para>
    </listitem>
   </varlistentry>
-  <varlistentry os="osuse">
+  <varlistentry>
    <term>Bug Reports</term>
    <listitem>
     <para>
-     To report bugs for &productname;, go to
-     <link xlink:href="https://bugzilla.opensuse.org/"/>, log in, and
-     click <guimenu>New</guimenu>.
+      To report documentation bugs in <phrase os="sles;sled">
+      <link xlink:href="https://bugzilla.suse.com/"/>
+     </phrase><phrase os="osuse">
+      <link xlink:href="https://bugzilla.opensuse.org/"/>
+     </phrase>, use the links next to each headline in the HTML version of this
+     document. <guimenu>Report Documentation Bug</guimenu> opens Bugzilla,
+     preselects the right product version and category, and adds a link to the
+     specific section. You can start typing your bug report right away.
+     (A Bugzilla account is required).
     </para>
    </listitem>
   </varlistentry>
-  <varlistentry os="sles;sled">
-   <term>User Comments</term>
+  <varlistentry>
+   <term>Mail</term>
    <listitem>
     <para>
-     We want to hear your comments about and suggestions for this manual and
-     the other documentation included with this product. Use the User Comments
-     feature at the bottom of each page in the online documentation or go to
-     <link xlink:href="http://www.suse.com/documentation/feedback.html"/> and
-     enter your comments there.
+     Alternatively, you can report errors and send feedback concerning the
+     documentation to <literal>doc-team@suse.com</literal>. Make sure to include the
+     document title, the product version and the publication date of the
+     documentation. Refer to the respective section number and title (or
+     include the URL) and provide a concise description of the problem.
     </para>
    </listitem>
   </varlistentry>
   <varlistentry>
-   <term>Mail</term>
+   <term>Contributions</term>
    <listitem>
     <para>
-     For feedback on the documentation of this product, you can also send a
-     mail to <literal>doc-team@suse.com</literal>. Make sure to include the
-     document title, the product version and the publication date of the
-     documentation. To report errors or suggest enhancements, provide a concise
-     description of the problem and refer to the respective section number and
-     page (or URL).
+     To contribute to this documentation, use the <guimenu>Edit Source</guimenu>
+     links next to each headline in the HTML version of this document. They
+     take you to the source code on GitHub, where you can open a pull request.
+     (A GitHub account is required).
```

Comment:
The parenthesis looks a bit strange to me. Maybe remove them?

```suggestion
     A GitHub account is required.
```

Reply:
Done.